src/main/java/nl/aerius/print/ChromeAttachment.java

```diff
@@ -1,25 +1,18 @@
 /*
- * The MIT License
+ * Copyright the State of the Netherlands
  *
- * Copyright 2018 Intuit Inc.
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
  *
- * Permission is hereby granted, free of charge, to any person obtaining a copy
- * of this software and associated documentation files (the "Software"), to deal
- * in the Software without restriction, including without limitation the rights
- * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
- * copies of the Software, and to permit persons to whom the Software is
- * furnished to do so, subject to the following conditions:
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
  *
- * The above copyright notice and this permission notice shall be included in
- * all copies or substantial portions of the Software.
- *
- * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
- * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
- * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
- * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
- * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
- * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
- * THE SOFTWARE.
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see http://www.gnu.org/licenses/.
  */
```

Comment:
This removes a license already present, which feels wrong. One for @JornC to check if this is okay.

Reply:
Yeah removing this MIT license and replacing it with our own violates the MIT license. We can keep this code in the project, but only if we include the MIT license. We have not written the code in this file, only modified it.

>  The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.

Reply:
I've reverted it. It looks to me the content is rather different from the original. Either way we need to exclude this file from spotless/sonar otherwise it will give errors or move it to a different namespace.

Reply:
I've rewritten the ChromeAttachment class to be ~completely original, and taken the liberty to push it into this PR @Hilbrand (aswell as accidentally push it to a new branch on this remote that I don't seem to be able to delete -_-), this class can legitimately have our GPL license, sidestepping the issue.